src/lib/security/ctrl_alt_del_config.rb

```diff
@@ -27,14 +27,14 @@ module CtrlAltDelConfig
     include Yast::Logger
     Yast.import "SCR"
     Yast.import "Arch"
-    Yast.import "Package"
+    Yast.import "PackageSystem"
     Yast.import "FileUtils"
 
     SYSTEMD_FILE = "/etc/systemd/system/ctrl-alt-del.target"
 
     class << self
       def systemd?
-        Yast::Package.Installed("systemd")
+        Yast::PackageSystem.Installed("systemd")
```

Comment:
I guess this change does not affect any other scenario, isn't it?

Reply:
I think it is quite save because it calls rpm directly.